server.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
+const app = express()
```

Comment:
It's a good practice to handle possible errors in your application. Consider adding an error handling middleware to your express app. [medium]